app/models/student.rb

```diff
@@ -50,21 +45,71 @@ class Student < ActiveRecord::Base
     presence: true
   )
 
+  name_pattern = /\A[^0-9`!@#\$%\^&*+_=]+\z/
+
+  validates(
+    :first_name,
+    :last_name,
+    length: {
+      minimum: 3,
+      maximum: 20,
+    },
+    format: {
+      with: name_pattern
```

Comment:
Nu folosi `name_pattern`, simplu adauga in locul lui `/\A[^0-9`!@#\$%\^&*+_=]+\z/`